tests/recipe/create_recipes.test.js

```diff
@@ -0,0 +1,114 @@
+const supertest = require("supertest");
+const app = require("../../app");
+const Recipe = require("../../models/recipe");
+const User = require("../../models/user");
+const api = supertest(app);
+const mongoose = require("mongoose");
+
+const {
+  initialUsers,
+  recipesInDb,
+  recipeToAdd,
+  recipeReview,
+} = require("../test_helper");
+const STATUS_CODE = require("../../shared/errorCode");
+
+const {
+  createUserAndLogin,
+  addRecipe,
+  addToRecipe,
+  getRecipe,
+  getUser,
+} = require("../api_test_helpers");
+
+const RECIPE_API = "api/recipe";
+
+let authHeader;
+let userId;
+let token;
+
+let recipeId;
```

Comment:
Global variables like this can be dangerous. Sometimes they change when you don't expect it. It's better to have local variables.